pages/common/docker-exec.md

```diff
@@ -22,3 +22,7 @@
 - Set an environment variable in a running bash session:
 
 `docker exec -it -e {{variable_name}}={{value}} {{container_name}} {{/bin/bash}}`
+
+- Execute command as a specific user:
```

Comment:
```suggestion
- Run a command as a specific user:
```

Reply:
I choose `execute` just because of `exec` in the command btw I agree with the `run` verb.

Reply:
Either would work here. I'm include to use execute, but run works too

Reply:
@sbrl Should we add the condition that the user should be already added to the container? Otherwsie we get this error message: `unable to find user XXX: no matching entries in passwd file`

Reply:
@navarroaxel @sbrl Could I change the example to this:
- Run a command as root user:
`docker exec --user {{root}} {{container_name}} {{command}}`

Reply:
I take back my words :)
As one of `tldr` user of I prefer the simple general example

Reply:
Also, I applied the requested change to code but now I do not resolve the conversation to reach finalize the sentence.

Reply:
what is the decision?